lib/config/cpm.cfg

```diff
@@ -15,7 +15,7 @@ CLIB    default -lcpm_clib -D__Z80__ -D__Z80 -Cc-standard-escape-chars -LDESTDIR
 CLIB    ixiy -lcpmixiy_clib -D__Z80__ -D__Z80 -mz80_ixiy -startuplib=z80iy_crt0 -Ca-IXIY -Cl-IXIY -Cc-standard-escape-chars -LDESTDIR/lib/clibs/ixiy
 CLIB    z180 -lcpmz180_clib -startuplib=z180_crt0 -Cc-standard-escape-chars -LDESTDIR/lib/clibs/z180 -LDESTDIR/lib/clibs/z180 -mz180 -D__Z180
 CLIB    8080 -lcpm8080_clib -Cc-standard-escape-chars -m8080 -D__8080 -startuplib=8080_crt0 -pragma-define:CLIB_DISABLE_FGETS_CURSOR=1 -LDESTDIR/lib/clibs/8080
-CLIB    8085 -lcpm8080_clib -Cc-standard-escape-chars -m8085 -D__8085 -startuplib=8085_crt0 -l8085_opt -pragma-define:CLIB_DISABLE_FGETS_CURSOR=1 -LDESTDIR/lib/clibs/8085
```

Comment:
Can you update +test as well?

Reply:
I think the +test target `config/test.cfg` already has a 8085 specific build, but I've fixed a bit of formatting on the way past.
Have I missed what you intended?

Reply:
Ah yeah, it's okay...just ignore me! :)